plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/WriteFunction.java

```diff
@@ -18,16 +18,12 @@
 
 public interface WriteFunction
 {
+    void setNull(PreparedStatement statement, int index) throws SQLException;
```

Comment:
I'd keep the ordering as it was originally - no pressing need to re-order them here (and it seemed wrong to me when I first saw it).